docs/data_types/master05-text_package.adoc

```diff
@@ -199,7 +199,7 @@ rather than a name. For example, in any `ENTRY` describing current problems, the
 the name/value pair "allergies: NONE". Here, "allergies" will be a `DV_CODED_TEXT`, and "NONE"
 will be either a `DV_CODED_TEXT` or a `DV_TEXT`; the two will be associated by a containing object,
 such as an instance of the `ELEMENT` class from the EHR RM. There is no explicit model of negation
-in openEHR.
+in the openEHR Reference Model; rather it's usually solved by specific 'exclusion' entry archetypes.
```

Comment:
in the openEHR Reference Model. Negation statements are normally solved by specific 'exclusion' archetypes.